docs/types/value-types.rst

```diff
@@ -59,15 +59,15 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand. The
-expression ``x << y`` is equivalent to ``x * 2**y``, and, for positive integers,
-``x >> y`` is equivalent to ``x / 2**y``. For negative ``x``, ``x >> y``
-is equivalent to dividing by a power of ``2`` while rounding down (towards negative infinity).
-Shifting by a negative amount throws a runtime exception.
+The result of a shift operation has the type of the left operand, truncating the result to match the type.
+
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values,  ``x >> y`` is equivalent to ``x / 2**y``, with values rounded down towards negative infinity.
```

Comment:
I'm still not too happy with this. If you say it is equivalent to `x/ 2**y`, then it should be, but `/` has a well-defined rounding semantics that is different to the one given here.

Reply:
Perhaps "similar to"?

Reply:
Ah, I think I understand what @chriseth said. For `x=1; y=1`, `x>>y` and `x/2**y` produce the same result (`=0`) but for `x=-1;y=1` `x>>y` produces `-1` while `x/2**y` produces `0`. So, drawing the equivalence would be problematic.

iiuc, perhaps we could say `x>>y` is equivalent to `x/2**y` for positive x, while it rounds down the result of `x/2**y` for negative `x`?

Edit: I just realized the text in red said something to this effect :)

Reply:
OK @bshastry and @chriseth while I was testing this I tried multiple values and as far as I could tell, the calculation (x/2**y) was the same for all values, but the difference was that with negative x values, the rounding is different?

My main issue with the original text was that as far as I could tell `dividing by a power of 2` was missing a lot of detail (divide what by two?, and there are two values, but only one mentioned etc.) and that with my experimentation, it was still x/2**y (but with the rounding difference).

So I think…

```suggestion
- For positive and negative ``x`` values,  ``x >> y`` is equivalent to ``x / 2**y``, with values rounded down towards negative infinity for negative ``x`` values.
```

Covers it? I take it that negative y values make no difference?

Reply:
What about using your original next and replacing "equivalent" by "similar"? My problem with the current formulation is that it might suggest that you can first do the regular division and then modify the value, but you actually have to do a different division instead.